frontend/works_shg_app/lib/pages/org_profile.dart

```diff
@@ -135,8 +127,7 @@ class _ORGProfilePage extends State<ORGProfilePage> {
                                         'PG_${e.orgAddress?.first.city?.toUpperCase()}') ??
                                     'NA',
                                 i18.common.ward: t.translate(
-                                        e.orgAddress?.first.boundaryCode ??
-                                            'NA') ??
+                                        'PG_CITY_${e.orgAddress?.first.boundaryCode ?? 'NA'}') ??
```

Comment:
for boundary PG_CITYA_ADMIN_B1
this is the format to localise

Reply:
Made the required changes